### DISCUSSION TITLE
NEW ICON PROPOSAL

### DISCUSSION THREAD

**Discussion by masterleo** [ImageMagick/ImageMagick#5273] [General]
### Is your feature request related to a problem? Please describe.

The pakage is installed with ubuntu stock install so it is very important for it to look nice. 
I propose a new one in a style named "flat icon"
here is the old, the new look and the icon in SVG format

### Describe the solution you'd like

![rect4894](https://user-images.githubusercontent.com/8157302/165067204-11b5edeb-0d76-4f00-be18-87e8175c0610.png)
![better](https://user-images.githubusercontent.com/8157302/165067282-42ca21e3-f5c0-42dd-94ee-3dfbb7dde1d7.png)
[hat magik.pdf](https://github.com/ImageMagick/ImageMagick/files/8553547/hat.magik.pdf)
![hat magik](https://user-images.githubusercontent.com/8157302/165067732-0fd14fcd-0225-42af-9409-a5c6ed483c8e.svg)

**Comment by kmgallahan** [comment]
The current icon is a work of art deserving of the Louvre. I hope it is never replaced.

**Comment by masterleo** [comment]
Absolutly ! Like the 300 years old painting that are stored there ! It does belong **there** but **not** in a leading linux disto. This program is not used by consumers I don't even know why it shows up in Ubuntu apps. Do you even use ubuntu server ? The icon is absolutly not a peace of art in term of Icon design in the year 2022 !! How can you say such a thing, at least remove this white backround no ? Do you see any other icon like this somewhere in the ubuntu default apps ? How old is the icon ?

**Comment by blue-devil-93** [comment]
I like the current icon more. Yours is good for some custom icon themes but not as an default icon on all distributions.
